pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[tool.black]
+line-length = 120
+skip-string-normalization = true
+target-version = ['py39']
+include = "src/vorta/.*.py$"
+
+[tool.isort]
+profile = "black"
+multi_line_output = 3
+lines_between_sections = 0
+src_paths = ["src", "tests"]
```

Comment:
As noted in their [docs](https://pycqa.github.io/isort/docs/configuration/black_compatibility.html) the parameters `skip_gitignore` and `line_length` should also be set.